netns_linux.go

```diff
@@ -1,4 +1,4 @@
-// +build linux
+// +build linux,go1.10
```

Comment:
Looks like go1.15 is now the latest officially stable version. I think it is safe to merge this one then.